spec/features/bars/administrator_can_delete_bar_spec.rb

```diff
@@ -35,4 +36,46 @@
 
     expect(page).to_not have_content("Delete")
   end
+
+  scenario 'bar is no longer in database' do
+    admin = FactoryGirl.create(:user, admin: true)
+    bar = FactoryGirl.create(:bar)
+
+    sign_in_as(admin)
+
+    visit bar_path(bar)
+
+    click_link("Delete")
+
+    expect(page).to_not have_content(bar.name)
+  end
+
+  scenario 'bar details page no longer exists' do
+    admin = FactoryGirl.create(:user, admin: true)
+    bar = FactoryGirl.create(:bar)
+
+    sign_in_as(admin)
+
+    visit bar_path(bar)
+
+    click_link("Delete")
+
+    expect(page).to_not have_content(bar.name)
+    expect(page).to_not have_content(bar.address)
+    expect(page).to_not have_content(bar.state)
+  end
```

Comment:
This test is incorrect. The bar details page wouldn't exist if you deleted the bar. How would you update this test to reflect that?